koku/masu/database/aws_report_db_accessor.py

```diff
@@ -275,9 +275,15 @@ def back_populate_ocp_infrastructure_costs(self, start_date, end_date, report_pe
         }
         self._prepare_and_execute_raw_sql_query(table_name, sql, sql_params)
 
-    def populate_ocp_on_aws_tag_information(self, bill_ids, start_date, end_date):
+    def populate_ocp_on_aws_tag_information(self, bill_ids, start_date, end_date, current_ocp_report_period_id):
```

Comment:
Why is this variable `current_ocp_report_period_id` and the rest are just `report_period_id`?

Reply:
Because I'm stupid obviously :smile: Thanks Michael!!
```suggestion
    def populate_ocp_on_aws_tag_information(self, bill_ids, start_date, end_date, report_period_id):
```